can_db/e46.md

```diff
@@ -4,24 +4,30 @@ The CAN bus is not available at the OBD-II port, so it must be tapped into behin
 The easiest places to do this are either behind the intrument cluster or at the steering angle sensor, 
 which is located at the base of the steering column near the firewall.
 
+![Steering Angle Sensor Plug](../images/e46_sensor_plug.jpg)
```

Comment:
Great photo, makes it clear where the harness is.

I just pushed 56a8228cba11c6db2ae00abbef45d314450e28d4 with a re-compressed version of that photo.
Sorry I grew up in dial-up time, and still trying to save a few kB when possible 😄
You can now remove the file from this PR.

Reply:
(please do keep the reference in the `.md` file though!)